### PR TITLE
Implemented reading RI from CSV data for PandasData

### LINKED ISSUE #30: Add reading retention indices from PandasData file types
Currently, the RI is only computed from carbon numbers if present. It should also be possible to read a column called 'RI', 'retention_index' etc. in order to read the values and initialize them.

### PR DESCRIPTION
**Pull request by hechth**
Fixes #30 

Also introduces multiple possible names for the RI column.

### CODE REVIEW

File: RIAssigner/data/PandasData.py [hunk 1]
```diff
@@ -44,13 +44,17 @@ def _init_rt_column_info(self):
     def _init_ri_column_info(self):
         """ Initialize retention index column name and set its position next to the retention time column. """
         self._ri_index = 'retention_index'
-        self._ri_position = self._rt_position + 1
+        if self._ri_index in self._data.columns:
+            self._ri_position = self._data.columns.get_loc(self._ri_index)
```
Comment (maximskorik) [inline RIAssigner/data/PandasData.py:49]: That looks nice, I've had no idea something like `get_loc` existed.

**Review by maximskorik — APPROVED** [review]

File: RIAssigner/data/PandasData.py (hunk 1 above)
Comment (hechth) [reply RIAssigner/data/PandasData.py:49]: This is only for Pandas data frames. Found it in the documentation [here](https://pandas.pydata.org/docs/reference/api/pandas.Index.get_loc.html#pandas.Index.get_loc).

Comment (maximskorik) [reply RIAssigner/data/PandasData.py:49]: yup, I checked it out already :)

File: RIAssigner/data/PandasData.py [hunk 2]
```diff
@@ -7,6 +7,7 @@
 class PandasData(Data):
     """ Class to handle data from filetypes which can be imported into a pandas dataframe. """
     _rt_column_names = set(['RT', 'rt', 'rts', 'retention_times', 'retention_time', 'retention', 'time'])
+    _ri_column_names = set(['RI', 'ri', 'ris', 'retention_indices', 'retention_index', 'kovats', 'retentionindex'])
```
Comment (martenson) [inline RIAssigner/data/PandasData.py:10]: let's start a new standard to unite them all🤣

**Pull request merged**